### PR TITLE
Ilchec

### PR DESCRIPTION
**Pull request by ilchec**
Ready to merge with master, I guess?

### CODE REVIEW

**Review by LinguList — COMMENTED** [review]
You submitted with `git add *`, right? TThis results in sttuff being submitted, which we don't want. So you should delete all of this `.egg-info`, which is your Python code.

The title  in metadata.json should have a genitive s: "Daniel et al.'s" 

Otherwise it looks fine!

**Comment by ilchec** [comment]
@LinguList done!

File: setup.py [hunk 1]
```diff
@@ -7,20 +7,20 @@
 
 
 setup(
-    name='lexibank_chechuroborrowing',
+    name='lexibank_daniellingua',
```
Comment (LinguList) [inline setup.py:10]: having a different name than the ID below daniellingua vs. daniellinguafrancas is not permitted...

**Review by LinguList — APPROVED** [review]
Please just make sure the name is the same as indicated in setup.py, then re-install the package, and re-run the code, ideally making a fresh virtual environment in Python, and specifying a version of glottolog, concepticon, and clts (even if you don't use them): 

```
$ cldfbench lexibank.makecldf daniellinguafrancas --glottolog-version=4.3 --concepticon-version=2.4 --clts-version=1.4.1
```
If this does not work, use `v1.4.1` instead of `1.4.1` for the versions.

**Comment by ilchec** [comment]
Here's what I did:
[create new conda venv]
python setup.py install
python lexibank_daniellinguafrancas.py install
cldfbench lexibank.makecldf daniellinguafrancas --glottolog-version=4.3 --concepticon-version=2.4 --clts-version=1.4.1
(not working)
cldfbench lexibank.makecldf daniellinguafrancas --glottolog-version=v4.3 --concepticon-version=v2.4 --clts-version=v1.4.1
(not working)

the error is the same in both cases (lot's of file logs, the last two lines are):
cmdline: git checkout v4.3
stderr: 'error: pathspec 'v4.3' did not match any file(s) known to git'

**Comment by ilchec** [comment]
Also, should it really be "daniellingua" or "daniellinguafrancas" in the setup? (neither works though)

**Comment by LinguList** [comment]
one name for all, that's what I meant: daniellinguafrancas everywhere, 
logically.

**Comment by ilchec** [comment]
Yes, then that's exactly how I did it, with the same name everywhere. Didn't work =(

**Comment by LinguList** [comment]
That's not the reason it failed, it is because you do not seem to have 
glottolog installed on your system. I'll look at this now.

**Comment by LinguList** [comment]
I just tested this: you have apparently not yet run your catconfig commando:

```
$ cldfbench catconfig
```

This assigns the glottolog repository (see Annika's post: https://calc.hypotheses.org/2225)

But it may also be that you should not use conda for your venv, but Python's internal venv machine.

And you may want to specify the path to the glottolog repository, by adding `--glottolog PATHTOGLOTTOLOG`

**Comment by LinguList** [comment]
On my system, it runs perfectly, I just checked this.

**Comment by ilchec** [comment]
@LinguList I pushed the new version, but I'm not sure that everything worked well

File: cldf/cldf-metadata.json [hunk 2]
```diff
@@ -0,0 +1,362 @@
...
+            "rdf:type": "prov:Entity",
+            "dc:title": "Repository",
+            "rdf:about": "git@github.com:ilchec/daniellinguafrancas",
+            "dc:created": "428a3d0"
+        },
+        {
+            "rdf:type": "prov:Entity",
+            "dc:title": "Glottolog",
+            "rdf:about": "https://github.com/glottolog/glottolog",
+            "dc:created": "v4.1-20-gea2ca69206"
```
Comment (LinguList) [inline cldf/cldf-metadata.json:23]: This shows that you apparently use a version of glottolog that is after 4.1, but not 4.3, so it also shows that the glottolog version was not specified when creating the cldf dataset. I guess, you should go to your glottolog repository and `git pull` to have the most recent version. Could you do so (also with conceepticon) and check again?

**Comment by ilchec** [comment]
Pushed the new version again. No idea why it worked this way, but I had to manually copy the "etc" and "raw" folders into the system folder that venv uses. I also updated glottolog, so it should be OK now.

**Comment by ilchec** [comment]
Okay, somehow it's still 4.1... I'll try to figure it out

**Pull request closed without merging**